wsd/COOLWSD.cpp

```diff
@@ -5814,6 +5814,8 @@ class COOLWSDServer
 
         // If we have any delaying work going on.
         Delay::dumpState(os);
+
+        COOLWSD::SavedClipboards->dumpState(os);
```

Comment:
Presumably we don't have to worry about some scenario where dumpState can be called after COOLWSD::main has called cleanup and SavedClipboards has been reset?